packages/nimble-components/src/rich-text/editor/testing/rich-text-editor-utils.ts

```diff
@@ -1,16 +1,20 @@
 import type { RichTextEditor } from '..';
```

Comment:
I don't think this file belongs in the /editor/testing which is publicly exported.

It doesn't seem to be used by anything except files in [/tests](https://github.com/search?q=repo%3Ani%2Fnimble%20rich-text-editor-utils&type=code) folders
And it is still coupled to imports from /tests folders

Maybe we can move it to /rich-text/models/tests?